test/util/StaticAsyncHandler.ts

```diff
@@ -0,0 +1,20 @@
+import { AsyncHandler } from '../../src/util/AsyncHandler';
+
+export class StaticAsyncHandler<TOut> implements AsyncHandler<any, TOut> {
+  private readonly canHandleStatic: boolean;
+
+  private readonly handleStatic: TOut;
+
+  public constructor (canHandleStatic: boolean, handleStatic: TOut) {
```

Comment:
I just notice this space before `(` 😱 
Is this enforced by the linter?
I would consider changing this rule, to save an extra space on your screen,
unless @RubenVerborgh has another opinion on this.

Same with the empty lines between the fields btw.

Reply:
OMG I'm shocked

Reply:
I had no idea this was one of the religious programming discussions :D. It's how the default linter settings are set. I'll fix it in a next commit after this one is merged. It is actually one of the rules where Prettier conflicted, [here](https://github.com/prettier/prettier/issues/3845) is some discussion between the people pro and con.